docs/AboutUs.md

```diff
@@ -5,7 +5,13 @@ title: About Us
 
 We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
 
-You can reach us at the email `seer[at]comp.nus.edu.sg`
+You may reach us through the following email addresses:
+
+Joe Teddy Yosua Sanjaya:
+Yu Bowei: e0425419@u.nus.edu
+Eddy Chu:
```

Comment:
Please kindly help update my email, and for the others @cupofjoee, @petrickjerico, @ducbinh2611   
Eddy Chu: e0418218@u.nus.edu
Thanks

Reply:
Noted, will do!